planner_cspace/src/planner_3d.cpp

```diff
@@ -2051,6 +2067,174 @@ class Planner3dNode
     return true;
   }
 
+  void updateTemporaryEscapeGoal(const Astar::Vec& start_grid)
+  {
+    if (!has_map_ || !has_goal_ || !has_start_ || !cost_estim_cache_created_)
```

Comment:
Forgot to remove `cost_estim_cache_created_` condition when I moved `cost_estim_cache_static_` update to this function